app/src/main/res/values-es/strings.xml

```diff
@@ -1,17 +1,44 @@
 <resources>
+    <string name="app_name">SDK de mapas y navegación para Android.</string>
+
     <string name="title_mock_navigation">Ejemplo de navegación</string>
     <string name="description_mock_navigation">Simula una sesión de navegación usando un ejemplo del motor de localización</string>
 
     <string name="title_off_route_detection">Apagar detección de ruta</string>
-    <string name="title_reroute">Enrutando</string>
+    <string name="description_off_route_detection">Emplea la clase \"Route Utils\" para determinar si un usuario se encuentra fuera de la ruta. </string>
+
+    <string name="title_reroute">Enrutado</string>
     <string name="description_reroute">Prueba la función de enrutación perteneciente al SDK de navegación</string>
 
     <string name="title_navigation_route_ui">Ruta del mapa de navegación</string>
-    <string name="description_navigation_route_ui">Dibuja un ruta en el mapa</string>
+    <string name="description_navigation_route_ui">Muestra diferentes rutas en el mapa de navegación.</string>
 
+    <string name="title_navigation_launcher">Motor de navegación.</string>
     <string name="description_navigation_launcher">Entra en la experiencia de la interfaz</string>
 
     <string name="title_end_navigation">Finaliza navegación</string>
+    <string name="description_end_navigation">Muestra como finalizar la navegación utilizando la vista de navegación.</string>
+
+    <string name="title_dual_navigation_map">Doble mapa de navegación.</string>
+    <string name="description_dual_navigation_map">Muestra como añadir vista de mapas y navegación en la misma disposición.</string>
+
+    <string name="title_waypoint_navigation">Punto de referencia de navegación.</string>
+    <string name="description_waypoint_navigation">Navegación con puntos de referencia entre destinos. </string>
```

Comment:
Are we OK adding an empty space `" "` at the end of the string? The root `strings.xml` doesn't include it. The same thing happens with `title_embedded_navigation` and `error_valid_route_not_found`.